src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -800,12 +856,18 @@ def check_error_and_log(error_regex, log_message_format):
       self.set_arg(fuzz_args, constants.CMPLOG_LEVEL_FLAG,
                    rand_cmplog_level(self.strategies))
 
+      android_params = []
+      if environment.is_android():
+        android_params = self.get_afl_environment_variables()
+        android_params.append(android.util.get_device_path(self.afl_fuzz_path))
+
       # Attempt to start the fuzzer.
       fuzz_result = self.run_and_wait(
-          additional_args=fuzz_args,
+          additional_args=android_params + fuzz_args,
           timeout=max_total_time,
           terminate_before_kill=True,
-          terminate_wait_time=self.SIGTERM_WAIT_TIME)
+          terminate_wait_time=self.SIGTERM_WAIT_TIME,
+      )
```

Comment:
Change this to:
```
if is_android():
  self.run_and_wait(...)
else:
  self.run_and_wait(...)
```
I don't like how there's an empty android_params argument I need to worry about here.